app/components/bali/calendar/preview.rb

```diff
@@ -42,6 +52,9 @@ def with_out_date
         end
       end
 
+      # Calendar with all week
```

Comment:
Is this a Calendar with weekend days?

Reply:
This configuration displays the calendar from Monday to Sunday (All Week)